src/libraries/System.Memory/tests/Span/Replace.T.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.SpanTests;
```

Comment:
Nit: I don't believe we use file-scoped namespaces anywhere else in this project.  Let's remain consistent.